moon/cfc_chat_transit/server/modules/connect.moon

```diff
@@ -14,7 +14,8 @@ ChatTransit.PlayerConnect = (data) =>
             SteamName: name
             SteamId: SteamIDTo64 steamId if steamId
 
-ChatTransit.PlayerInitialSpawn = (ply) =>
+ChatTransit.PlayerInitialSpawn = (ply, mapTransition) =>
+    return if mapTransition
```

Comment:
Shouldn't we keep it how it is? For logging purposes, it could be useful to keep around...